ITEM.tsv

```diff
@@ -2962,13 +2962,13 @@ ITEM_20150317_002961	Mersumma Thorn
 ITEM_20150317_002962	Thronball Thorn 
 ITEM_20150317_002963	Cyst leaves 
 ITEM_20150317_002964	(Temporary) Yellow Cyst Leaves 
-ITEM_20150317_002965	Infro Holder Horn 
-ITEM_20150317_002966	Ingredient for crafting shards of memory. It is very thick. 
+ITEM_20150317_002965	$Infroholder Horn 
+ITEM_20150317_002966	$Ingredient for crafting the Fragment of Memory. It is very thick. 
```

Comment:
the Fragment of **a** Memory? (If it's an item, that would probably need change too)